perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/TasksManagerMethod.java

```diff
@@ -326,6 +326,31 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 			}
 			return null;
 		}
+	},
+
+	/*#
+	 * Stops dispatcher from propagating waiting tasks to the engine.
+	 * Tasks which were sent to the engine before won't be affected and will be finished.
+	 */
+	suspendTasksPropagation {
```

Comment:
Please add `parms.stateChangingCheck();` to bothe methods, since we want to be sure they are not called using GET.